From 4b9d701fabff8e7969db081406d00fa9fe46b3fd Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Thu, 27 Sep 2018 11:05:51 +0800
Subject: [PATCH] fix do_package_qa error

fix below error:
do_package_qa: QA Issue: ELF binary 'xxx/usr/sbin/registry' 
has relocations in .text [textrel]

Upstream-Status: Inappropriate [oe-specific]

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 47b8f1d..13b0f92 100644
--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,7 @@ GODEP=$(shell which godep || echo '')
 
 ${PREFIX}/bin/registry: $(GOFILES)
 	@echo "+ $@"
-	@go build -tags "${DOCKER_BUILDTAGS}" -o $@ ${GO_LDFLAGS}  ${GO_GCFLAGS} ./cmd/registry
+	@go build -tags "${DOCKER_BUILDTAGS}" -o $@ ${GO_LDFLAGS} ${GOBUILDFLAGS} ${GO_GCFLAGS} ./cmd/registry
 
 ${PREFIX}/bin/digest:  $(GOFILES)
 	@echo "+ $@"
-- 
2.7.4

